<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DatabaseLogIn</class>
 <widget class="QDialog" name="DatabaseLogIn">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>438</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/loginDatabaseScreen.jpg);</string>
  </property>
  <widget class="QLabel" name="label_Userid">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black;
font: 900 12pt &quot;Segoe UI Black&quot;</string>
   </property>
   <property name="text">
    <string>Enter Username: </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_password">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black;
font: 900 12pt &quot;Segoe UI Black&quot;</string>
   </property>
   <property name="text">
    <string>Enter Password: </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Userid">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>350</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_databasePass">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>350</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_databaseName">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>10</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: grey;
font: 900 20pt &quot;Segoe UI Black&quot;</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_error">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>200</y>
     <width>231</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black;</string>
   </property>
   <property name="text">
    <string>Username or password is incorrect!</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_error_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>200</y>
     <width>231</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black;</string>
   </property>
   <property name="text">
    <string>All fields must be filled!</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>389</x>
     <y>150</y>
     <width>40</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Show</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>270</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>270</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
